src/lib.rs

```diff
@@ -333,6 +347,22 @@ mod tests {
     }
 
     #[test]
+    fn iter_random_add() {
+        use rand::Rng;
+        let mut set = BitSet::new();
+        let mut rng = ::rand::weak_rng();
```

Comment:
let's be more consistent here - since we have a `use rand::` statement on top, let's include `weak_rng` there as well.